src/test/java/Lesson15/OrderDaoImplIntegrationTest.java

```diff
@@ -5,17 +5,54 @@
 
 import java.math.BigDecimal;
 import java.sql.SQLException;
+import java.util.Set;
```

Comment:
Orders and offices test must be in different classes. Also good test in our case cheack that both orders methods (getAll) return same resulr

Reply:
But i have this test. I compare their size. Not always sql result sorted in the same way.